exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,23 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function createAShoutyMessage(message) {
+	return (message.toUpperCase());
+}
+
+function shoutName(name) {
+	return (name.toUpperCase());
+}
+
+greeting = "Finally you're here! Come on in, ";
+
+function createAShoutyMessageWithName(message, name) {
+	return (createAShoutyMessage(message) + shoutName(name));
```

Comment:
Hi Temwa,

Great work, just a suggestion, u could have used the same function, e.g shoutName  perhaps just named differently to capitalise the shouty message.